wdl/transforms/new-base/src/main/scala/wdl/transforms/base/linking/expression/files/LiteralEvaluators.scala

```diff
@@ -107,4 +109,24 @@ object LiteralEvaluators {
        a.right.evaluateFilesNeededToEvaluate(inputs, ioFunctionSet, coerceTo)
       ) mapN { _ ++ _ }
   }
+
+  implicit val stringExpressionEvaluator: FileEvaluator[StringExpression] = new FileEvaluator[StringExpression] {
+    override def predictFilesNeededToEvaluate(a: StringExpression,
+                                              inputs: Map[String, WomValue],
+                                              ioFunctionSet: IoFunctionSet,
+                                              coerceTo: WomType
+    )(implicit
+      fileEvaluator: FileEvaluator[ExpressionElement],
+      valueEvaluator: ValueEvaluator[ExpressionElement]
+    ): ErrorOr[Set[WomFile]] = {
+      val predictedFiles: Seq[ErrorOr[Set[WomFile]]] = a.pieces.map {
+        case literal: StringLiteral =>
+          stringLiteralEvaluator.predictFilesNeededToEvaluate(literal, inputs, ioFunctionSet, coerceTo)
+        case _ => Valid(Set.empty[WomFile])
+      }
+      val sets: Set[WomFile] = predictedFiles.collect { case Valid(set) => set }.foldLeft(Set[WomFile]())(_ ++ _)
+      val errors: Seq[NonEmptyList[String]] = predictedFiles.collect { case Invalid(error) => error }
+      errors.headOption.map(head => head.invalid).getOrElse(sets.validNel)
+    }
+  }
```

Comment:
Can we add some unit testing of this? Looks like we already have a `FileEvaluatorSpec`.